app/controllers/friendships_controller.rb

```diff
@@ -12,16 +13,26 @@ def create
   end
 
   def update
+    @friend = Friendship.find(params[:id])
+    @ft = Friendship.create
+    @ft.user_id = current_user.id
+    @ft.friend_id = @friend.user_id
+    @ft.status = true
+    @ft.save
```

Comment:
- [ ] Like the last reviewer said, you should only save the reversed friendship after confirming that you managed to update the status of the first request to true.
What happens if there's an error updating @friendship and you already saved @ft?

First update @friendship and if that works you can save @ft.